packages/mui-codemod/README.md

```diff
@@ -2003,6 +2004,47 @@ npx @mui/codemod@latest deprecations/table-sort-label-classes <path>
 npx @mui/codemod@latest deprecations/typography-props <path>
 ```
 
+### v7.0.0
+
+#### `lab-removed-components`
+
+<!-- #npm-tag-reference -->
+
+```bash
+npx @mui/codemod@next v7.0.0/lab-removed-components <path>
+```
+
+Update the import of the following components and hook moved from `@mui/lab` to `@mui/material`:
+
+- Alert
+- AlertTitle
+- Autocomplete
+- AvatarGroup
+- Pagination
+- PaginationItem
+- Rating
+- Skeleton
+- SpeedDial
+- SpeedDialAction
+- SpeedDialIcon
+- ToggleButton
+- ToggleButtonGroup
+- usePagination
+
+It updates barrel file imports:
```

Comment:
I think this is more correct, plus is aligned with our [tree-shaking docs](https://mui.com/material-ui/guides/minimizing-bundle-size/#when-and-how-to-use-tree-shaking).

```suggestion
It updates named imports:
```

Reply:
```suggestion
It updates named imports from top-level `@mui/lab`:
```